os_builders/roles/vm_baseline/tasks/wazuh.yml

```diff
@@ -16,6 +16,11 @@
         state: present
         allow_downgrade: true
         update_cache: yes
+
+    - name: Remove Wazuh repo
+      ansible.builtin.apt_repository:
+        repo: "deb https://packages.wazuh.com/4.x/apt/ stable main"
+        state: absent
```

Comment:
Would this prevent users from updating the package later on?
Just thinking if there's a CVE against it, we'd need to get users to either replace their image or manually add this apt repo again to pull in any updates

Reply:
Discussed offline, if the version upgrades beyond our server version this will cause problems
Currently apt isn't fully respecting version pinning so lets leave this out for the moment